src/deadline_worker_agent/sessions/actions/scripts/attachment_download.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+#! /usr/bin/env python3
+import argparse
+import time
+import os
+import boto3
+
+from deadline.job_attachments import api
+
+"""
+A small script to download job output. Can provide just the Job ID to download all outputs
+for a Job, optionally include the Step ID to get all outputs for the Job's Step, or optionally
+include the Job, Step, and Task ID to get the outputs for a specific Task.
+
+Example usage:
+
+python attachment_download.py \
+    -pm /sessions/session-f63c206fb5f04c04aa17821001aa3847fajfm5x4/path_mapping.json \
+    -s3 s3://test-job-attachment/DeadlineCloud \
+    -m /sessions/session-f63c206fb5f04c04aa17821001aa3847fajfm5x4/manifests/0bb7eb91fdf8780c4a7e6174de6dfc5e_manifest \
+    -m /sessions/session-f63c206fb5f04c04aa17821001aa3847fajfm5x4/manifests/0bb7eb91fdf8780c4a7e6174de6dfc5e_manifest
+"""
+
+
+def download(s3_root_uri: str, path_mapping_rules: str, manifests: list[str]) -> None:
+
+    api.attachment_download(
+        manifests=manifests,
+        s3_root_uri=s3_root_uri,
+        boto3_session=boto3.session.Session(profile_name=os.environ.get("AWS_PROFILE")),
+        path_mapping_rules=path_mapping_rules,
+    )
+
+
+if __name__ == "__main__":
```

Comment:
Should we add graceful error handling to this? The default behavior of Python is to output a stack trace and this may not be the most customer-friendly way to present errors.

Same suggestion applies to `attachment_upload.py`.

Reply:
I'd like to take this in a follow-up PR, there are other error handling cases on my list that I'd like to address together.